templates/azure/index.html

```diff
@@ -249,29 +172,134 @@ <h2>Get Ubuntu Pro on Azure now</h2>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row--25-75 p-section--shallow">
+        <div class="col">
+          {{ image(url="https://assets.ubuntu.com/v1/42cc2819-ron-updated.png",
+                    alt="Ron Abellera",
+                    width="852",
+                    height="847",
+                    hi_def=True,
+                    loading="lazy") | safe
+          }}
+        </div>
+        <div class="col">
+          <div class="row">
+            <hr class="p-rule--muted" />
+            <div class="col-6">
+              <p class="p-heading--4"><em>"Canonical has proven to be an exceptional and influential partner in the Linux and open-source community. Our collaborative efforts with their engineering teams are driving innovative advancements in AI, security, and secure software supply chain. This partnership enables customers to seamlessly migrate their legacy on-premises distributions to a more secure environment with Ubuntu on Azure."</em></p>
+            </div>
+            <div class="col-3">
+              <p class="u-no-margin--bottom"><strong>Ron Abellera</strong></p>
+              <p class="u-text--muted">Global Black Belt in Linux at Microsoft</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
     <div class="row--50-50">
       <hr class="p-rule--muted" />
+      <div class="col">
+        <h2>Secure Data at Run-Time with Confidential Computing on Azure</h2>
+      </div>
+      <div class="col">
+        <p>If you're running critical workloads and need to process sensitive data, security at system memory is essential. With confidential computing, your workloads are  encrypted at run time, and are thus secure from all software vulnerabilities outside the VM perimeter - including the hypervisor and from unauthorized cloud admins. Pair confidential VMs with Ubuntu Pro for security in and out of your VM.</p>
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Secure during compute</li>
+          <li class="p-list__item is-ticked">Enables zero trust</li>
+          <li class="p-list__item is-ticked">Process sensitive workloads without worry</li>
+          <li class="p-list__item is-ticked">Available on Ubuntu Pro for added security or the free versions of Ubuntu</li>
+        </ul>
+        <hr class="p-rule--muted" />
+        <p><a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-confidential-vm-jammy?tab=Overview" class="p-button">Launch an Ubuntu Pro CVM now</a></p>
+        <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.ubuntu-24_04-lts?tab=Overview" style="margin-right: 1rem">Launch an Ubuntu CVM&nbsp;&rsaquo;</a>
+        <a href="/confidential-computing">Learn more about CVMs&nbsp;&rsquo;</a>
```

Comment:
You could use the [Vanilla cta block](https://vanillaframework.io/docs/patterns/cta-block) here

Reply:
Doesn't work in this case because of the top link

Reply:
Done